tests/fixtures/direct_award_project_services_fixture.json

```diff
@@ -0,0 +1,181 @@
+{
+  "links": {
+    "self": "http://localhost:5000/direct-award/projects/1/services"
+  },
+  "meta": {
+    "total": 9
+  },
+  "services": [
+    {
+      "data": {
+        "serviceName": "Service name",
+        "serviceDescription": "This is a really interesting and appropriate service description.",
+        "priceMin": "1.00",
+        "priceMax": "10.00",
+        "priceInterval": "hour",
+        "priceUnit": "virtual machine"
+      },
+      "id": "123456789",
+      "supplier": {
+        "contact": {
+          "email": "test@fixture.com",
+          "name": "Supplier contact name",
+          "phone": "01234 000 000"
+        },
+        "name": "Supplier name"
+      }
+    },
+    {
```

Comment:
did we really need this many?

Reply:
Probably not. But I am at the end of my tether with mocks and testing now... please don't make me touch it again for a bit.

Reply:
Probably not. But I am at the end of my tether with mocks and testing now... please don't make me touch it again for a bit.